pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSchemaRestletResource.java

```diff
@@ -318,4 +324,19 @@ private void deleteSchemaInternal(String schemaName) {
           Response.Status.INTERNAL_SERVER_ERROR);
     }
   }
+
+  private List<TableConfig> getTableConfigsForSchema(@Nullable String schemaName) {
```

Comment:
`schemaName` should never be null

Reply:
Wondering if we can move this method into `PinotHelixResourceManager`?

Reply:
yes, done